src/sql/parts/modelComponents/formLayout.css

```diff
@@ -0,0 +1,16 @@
+
+.model-table {
+	width:400px;
```

Comment:
where did this width come from?

Reply:
I'll change it to get from extension. for now it's hardcoded

---
In reply to: [184465342](https://github.com/Microsoft/sqlopsstudio/pull/1269#discussion_r184465342) [](ancestors = 184465342)